packages/monaco/src/browser/textmate/textmate-tokenizer.ts

```diff
@@ -39,6 +39,10 @@ export function createTextmateTokenizer(grammar: IGrammar): monaco.languages.Tok
     return {
         getInitialState: () => new State(INITIAL),
         tokenize(line: string, state: State) {
+            if (line.length > 1000) {
```

Comment:
I see many `*.min.js`files with a max line length of 500. So I think we should reduce this to `400`.

Reply:
I'm okay with 400. Just, it _feels_ like it could work with 1000, too.

Okay, let it be 400. We can try and change if needed.

Reply:
I've updated the limit of line length.